contracts/meta-oracles/lib/RSILibrary.sol

```diff
@@ -0,0 +1,84 @@
+/*
+    Copyright 2019 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity 0.5.7;
+
+import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+
+/**
+ * @title RSILibrary
+ * @author Set Protocol
+ *
+ * Library for calculating the Relative Strength Index
+ * 
+ */
+library RSILibrary{
+
+    using SafeMath for uint256;
+
+    /*
+     * Calculates the new relative strength index value using
+     * RSI time period, and the time series feed instance.
+     *
+     * RSI = 100 − 100 / 
+     *       (1 + (Daily Average Gain / Daily Average Loss)
+     *
+     * Daily Price Difference = Price(N) - Price(N-1) where N is number of days
+     * Daily Average Gain = Sum(Positive Daily Price Difference) / N 
+     * Daily Average Loss = -1 * Sum(Positive Daily Price Difference) / N 
+     * 
+     *
+     * Our implementation is simplified to the following for efficiency
+     * RSI = 100 - (100 * SUM(Loss) / ((SUM(Loss) + SUM(Gain)))
+     * 
```

Comment:
I know its not on the other EMA file, but let's add documentation here what `a`, `b`, and `c` below map to in this formula.